vm/src/tests/run_deprecated_contract_class_simplified.rs

```diff
@@ -16,6 +16,7 @@
 */
 use crate::stdlib::{collections::HashMap, prelude::*};
 
+use crate::stdlib::any::Any;
```

Comment:
Great @YairVaknin-starkware! Could you remove this unused dependency then?

Reply:
Done